code/modules/racial_roundstartperks/perk_h_bravado.dm

```diff
@@ -0,0 +1,178 @@
+
+/datum/perk/battleroar/assign(mob/living/carbon/human/H)
+	..()
+	holder.mod_climb_delay += 0.45
+	holder.falls_mod += 0.3
+
+/datum/perk/battleroar/remove()
+	holder.mod_climb_delay -= 0.45
+	holder.falls_mod -= 0.3
+	..()
+
+/datum/perk/battleroar/activate()
+	var/mob/living/carbon/human/user = usr
+	var/list/people_around = list()
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("You cannot muster your throat to have a roar just yet."))
+		return FALSE
+	cooldown_time = world.time + 15 MINUTES
+	log_and_message_admins("used their [src] perk.")
+	for(var/mob/living/carbon/human/H in view(user))
+		if(H != user && !isdeaf(H))
+			people_around.Add(H)
+	if(people_around.len > 0)
+		for(var/mob/living/carbon/human/participant in people_around)
+			to_chat(participant, SPAN_NOTICE("You feel a urge to fight by a powerful roar!"))
+			give_boost(participant)
+	give_boost(usr)
+	usr.emote("bellow")
+	return ..()
+
+/datum/perk/battleroar/proc/give_boost(mob/living/carbon/human/participant)
+	var/effect_time = 10 MINUTES //Stat boost 10 minutes! Only five points so its fair!
+	var/amount = 10
+	var/list/stats_to_boost = list(STAT_ROB = 5, STAT_TGH = 5, STAT_VIG = 5)
+	for(var/stat in stats_to_boost)
+		participant.stats.changeStat(stat, amount)
+		addtimer(CALLBACK(src, .proc/take_boost, participant, stat, amount), effect_time)
+
+/datum/perk/battleroar/proc/take_boost(mob/living/carbon/human/participant, stat, amount)
+	participant.stats.changeStat(stat, -amount)
+
+
+
+/datum/reagent/medicine/hakronol
+	name = "Hakronol"
+	id = "hakronol"
+	description = "A chemical naturally produced by several species that learned to push to their body to its limits, allowing them to possibly recover from mild injuries."
+	taste_description = "grossness and pain"
+	reagent_state = LIQUID
+	color = "#8040FF"
+	nerve_system_accumulations = 45
+	appear_in_default_catalog = TRUE
+	constant_metabolism = TRUE
+	scannable = TRUE
+
+/datum/reagent/medicine/hakronol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.adjustOxyLoss(-0.2)
+	M.heal_organ_damage(0.015, 0.015)
+	M.adjustToxLoss(-0.1)
+	M.add_chemical_effect(CE_BLOODCLOT, 0.01)
+	M.add_chemical_effect(CE_PAINKILLER, 10, TRUE)
+
+/datum/reagent/medicine/hustikol
+	name = "Hustikol"
+	id = "hustikol"
+	description = "A chemical naturally produced by humans pushed to their limit. Induces a recovery coma to heal their wounds."
+	taste_description = "bitterness and agony"
+	reagent_state = LIQUID
+	color = "#ded890"
+	nerve_system_accumulations = 45
+	appear_in_default_catalog = TRUE
+	constant_metabolism = TRUE
+	scannable = TRUE
+
+/datum/reagent/medicine/hustikol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.heal_organ_damage(0.02, 0.1, 2) //Trade Healing damage to  |
+	M.add_chemical_effect(CE_BLOODCLOT, min(4,2.6)) // Clotting V
+	M.adjustOxyLoss(-0.3) // Heart start beating faster. You get more oxygen to your limbs and or gans
```

Comment:
Heart's not beating faste if you add CE_STABLE and then CE_PULSE -3, that's literally the heart beating **SLOWER** btw